### PR TITLE
Krystian Podemski's interview

### PR DESCRIPTION
**Pull request by Julievrz**
I'd like to publish the ITW on January 18, when I come back from university. In the meantime, feel free to review the article and make some suggestions. :-)

### CODE REVIEW

File: news/_posts/2022-01-18-contributor-interview-krystian-podemski.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "From contributor to maintainer: Krystian Podemski's experience with the PrestaShop project"
+subtitle: "Discover the faces behind the commits"
+date:   2022-01-18 
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2021/01/krystian-podemski.jpg
```
Comment (MatShir) [inline news/_posts/2022-01-18-contributor-interview-krystian-podemski.md]:
```suggestion
image: /assets/images/2022/01/krystian-podemski.jpg
```
I suppose ^^

**Review by MatShir — APPROVED** [review]
👏  very nice interview ! And thanks for @kpodemski for taking the time !

File: news/_posts/2022-01-18-contributor-interview-krystian-podemski.md (hunk 1 above)
Comment (Julievrz) [reply news/_posts/2022-01-18-contributor-interview-krystian-podemski.md]: Second time in a week... guess I'm stuck in 2021. Not the best year to be stuck though 😅

**Review by matks — APPROVED** [review]
👍

**Review by atomiix — APPROVED** [review]

**Comment by ghost** [comment]
Why not an image with the good ratio ?

krystian-podemski

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Julievrz

**Pull request merged**